pyaedt/edb_grpc/core/edb_data/layer_data.py

```diff
@@ -0,0 +1,1753 @@
+from __future__ import absolute_import
+
+from collections import OrderedDict
+import re
+import time
+
+from pyaedt import pyaedt_function_handler
+from pyaedt.generic.clr_module import List
+
+
+class EDBLayer(object):
+    """Manages EDB functionalities for a layer.
+
+    .. deprecated:: 0.6.5
+        There is no need to use core_stackup anymore. You can instantiate new class stackup directly from edb class.
```

Comment:
```suggestion
        There is no need to use ``core_stackup`` anymore. You can instantiate a new stackup directly from the ``EDB`` class.
```
This wording might still need some improvement--and changes should be made everywhere this deprecation notice exists.